UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.LoadTests/TestCase/ESSPTUsingLiveLogsWithCachedReadMe.js

```diff
@@ -0,0 +1,142 @@
+                { duration: stageTime, target: testStages[13] },
+                { duration: stageTime, target: testStages[14] },
+                { duration: stageTime, target: testStages[15] },
+                { duration: stageTime, target: testStages[16] },
+
+                { duration: stageTime, target: testStages[17] },
+                { duration: stageTime, target: testStages[18] },
+                { duration: stageTime, target: testStages[19] },
+                { duration: stageTime, target: testStages[20] },
+                { duration: stageTime, target: testStages[21] },
+                { duration: stageTime, target: testStages[22] },
+                { duration: stageTime, target: testStages[23] },
+                { duration: stageTime, target: testStages[24] },
+                { duration: stageTime, target: testStages[25] },
+                { duration: stageTime, target: testStages[26] },
+
+                { duration: stageTime, target: testStages[27] },
+                { duration: stageTime, target: testStages[28] },
+                { duration: stageTime, target: testStages[29] },
+                { duration: stageTime, target: testStages[30] },
+                { duration: stageTime, target: testStages[31] },
+                { duration: stageTime, target: testStages[32] },
+                { duration: stageTime, target: testStages[33] },
+                { duration: stageTime, target: testStages[34] },
+                { duration: stageTime, target: testStages[35] },
+                { duration: stageTime, target: testStages[36] },
+
+                { duration: stageTime, target: testStages[37] },
+                { duration: stageTime, target: testStages[38] },
+                { duration: stageTime, target: testStages[39] },
+                { duration: stageTime, target: testStages[40] },
+                { duration: stageTime, target: testStages[41] },
+                { duration: stageTime, target: testStages[42] },
+                { duration: stageTime, target: testStages[43] },
+                { duration: stageTime, target: testStages[44] },
+                { duration: stageTime, target: testStages[45] },
+                { duration: stageTime, target: testStages[46] },
+
+                { duration: stageTime, target: testStages[47] },
+                { duration: stageTime, target: testStages[48] },
+                { duration: stageTime, target: testStages[49] },
+                { duration: stageTime, target: testStages[50] },
+                { duration: stageTime, target: testStages[51] },
+                { duration: stageTime, target: testStages[52] },
+                { duration: stageTime, target: testStages[53] },
+                { duration: stageTime, target: testStages[54] },
+                { duration: stageTime, target: testStages[55] }
+            ],
+      },
+      'invalidate-ReadMe':{
+        exec:'invalidateReadMeCache',
+        executor: 'shared-iterations',
```

Comment:
Why is this a shared-iterations exectutor?  That doesn't really make sense if there's on 1 VU.  Consider using per-vu-iterations as that is less confusing

Reply:
Resolved